appkit/react-native/core/migration-multichain.mdx

```diff
@@ -0,0 +1,329 @@
+---
+title: Migrate to Multichain
+description: A guide to help you migrate your existing Reown AppKit React Native integration to the new multichain version.
+---
+
+import { Card, CardGroup } from '@mintlify/components';
+
+This guide will walk you through the necessary steps to update your existing Reown AppKit React Native integration to the latest multichain version. The new version introduces a more flexible and powerful architecture, centered around a core AppKit library and distinct chain-specific adapters.
+
+## Key Changes at a Glance
+
+Before diving into the specific steps, here's a high-level overview of the most significant changes:
+
+-   **Core Library & Adapters:** Instead of using monolithic, EVM-specific packages like `@reown/appkit-wagmi-react-native` or `@reown/appkit-ethers-react-native` as your primary entry point, you will now use a core package `@reown/appkit-react-native`. Chain-specific logic for EVM (Ethers, Wagmi), Solana, Bitcoin, and potentially other chains is handled by separate, installable adapter packages (e.g., `@reown/appkit-ethers-react-native`, `@reown/appkit-solana-react-native`).
```

Comment:
```suggestion
-   **Core Library & Adapters:** Instead of using monolithic, EVM-specific packages like `@reown/appkit-wagmi-react-native` or `@reown/appkit-ethers-react-native` as your primary entry point, you will now use a core package `@reown/appkit-react-native`. Chain-specific logic for **EVM (Ethers, Wagmi), Solana, Bitcoin**, and potentially other chains is handled by separate, installable adapter packages (e.g., `@reown/appkit-ethers-react-native`, `@reown/appkit-solana-react-native`).
```